.github/workflows/release-charts.yaml

```diff
@@ -0,0 +1,106 @@
+name: Release Helm Charts
+
+on:
+  workflow_dispatch:
```

Comment:
The expectation is that after merging the PR that was generated by the action above, the release manager will go and trigger this action manually. I think that is completely fine for now, I am just confirming.

Reply:
Yes, correct. Even though we can run it on every PR to main (or if it's merge common from release branch) having more control is preferable for now. So, manual run by a release engineer.